src/services/balances.rs

```diff
@@ -45,6 +46,14 @@ pub fn balances(
         })
         .collect();
 
+    service_balances.sort_by(|a, b| {
+        a.fiat_balance
+            .parse::<f64>()
+            .unwrap_or(0.0)
+            .partial_cmp(&b.fiat_balance.parse::<f64>().unwrap_or(0.0))
+            .unwrap_or(Ordering::Equal)
+    });
+    service_balances.reverse();
```

Comment:
Wouldn't it be easier to just swap `a` and `b` in the `sort_by`?

Reply:
absolutely, well spotted.